ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/AttestationValidator.java

```diff
@@ -174,14 +176,18 @@ SafeFuture<InternalValidationResultWithState> singleOrAggregateAttestationChecks
 
               if (!attestation.isSingleAttestation()) {
                 // [REJECT] The number of aggregation bits matches the committee size
```

Comment:
I'd take the opportunity here to move the comment to the `if` check so it continues to matche what the code is doing.

Reply:
but this whole block only does that... i was just making it not go boom... not sure it'll be v. clear if its down 2 levels?